src/style.css

```diff
@@ -1,3 +1,89 @@
-body {
+/* #region ZERO */
+body{
+  padding: 0;
   margin: 0;
+  font-family: Roboto, sans-serif;
+  padding-bottom: 2000px;
+}
+
+.wrapper{
+  max-width: 1200px;
```

Comment:
Don't fix container size (if there is no such a requirement). Let the content size dictate it to avoid overflow or accidental scroll bar. If there's no specific requirement for a `max-width`, consider using a more responsive approach.